packages/surf/package.json

```diff
@@ -0,0 +1,69 @@
+{
+    "name": "@orchestrator-ui/surf",
```

Comment:
Suggest to reflect this naming in the folder.

As it's also done in the orchestrator-ui package.

Or when you don't pulbish to NPM (which we won't do for SURF???) -> we could just use the name "surf"

Reply:
- Can this new "surf" package use library functions (e.g. stuff from @orchestrator-ui)
- How big will the build of this package be?

@ricardovdheijden / @DutchBen : I don't know if it's a good idea to now have a new "wfo-surf-ui" top level app. I saw it's already merged; but this leads to a lot of duplication and extra maintenance: you now have to sync both folders. At first, till march, this will be only one extra route: "/tickets" which we could also just hide based on an env var.

Personally I would've tried to avoid having 2 duplicate apps as long as possible...

So something to keep in mind when reviewing code. Now commits in "wfo-ui" should be duplicated in "wfo-surf-ui"!

Reply:
My 2 cents are that because we have this package setup changes that need to be replicated will be very rare or at least rarer than they used to be but good point to keep in mind. I am happy to see that it the setup seems to work.